controllers/user.js

```diff
@@ -453,3 +453,87 @@ exports.postForgot = function(req, res, next) {
     res.redirect('/forgot');
   });
 };
+
+/**
+ * GET /org/edit-question
+ * Edit the selected question
+ */ 
+exports.getEditQuestion = function(req, res) {
+  if (req.isAuthenticated()) {
+    res.render('account/org/edit-question', {
+    title: 'Edit Question'
+    });
+  }
+};
+
+/**
+ * POST /org/edit-question
+ * Apply changes to the question and return to the edit-survey page
+ */ 
+exports.postEditQuestion = function(req, res) {
+  if (req.isAuthenticated()) {
+    res.render('account/org/edit-survey', {
+    title: 'Edit Survey'
+    });
+  }
+};
+
+/**
+ * GET /edit-survey
+ * Forgot editSurvey page.
+ */
+exports.getEditSurvey = function(req, res) {
+  if (req.isAuthenticated()) {
+    res.render('account/org/edit-survey', {
+    title: 'Edit Survey'
```

Comment:
I finally understand your other indentation remarks. hahaha sorry about that. Pushed 

    On Sunday, February 7, 2016 2:21 PM, DylanMoz <notifications@github.com> wrote:
 
 

 In controllers/user.js:> +exports.postEditQuestion = function(req, res) {
> +  if (req.isAuthenticated()) {
> +    res.render('account/org/edit-survey', {
> +    title: 'Edit Survey'
> +    });
> +  }
> +};
> +
> +/**
> + * GET /edit-survey
> + * Forgot editSurvey page.
> + */
> +exports.getEditSurvey = function(req, res) {
> +  if (req.isAuthenticated()) {
> +    res.render('account/org/edit-survey', {
> +    title: 'Edit Survey'
indendation is off. title:... should be tabbed in once more—
Reply to this email directly or view it on GitHub.